### ISSUE TITLE
Wallet crash above v3.5.9.5+

### ISSUE THREAD

**Issue by AndreasMager** [gridcoin-community/Gridcoin-Research#461]
System:
Win 10
 v3.5.9.5-7

My wallet seems to go offline after a por event.

**Comment by iFoggz** [comment]
unfortunately we are gonna need more information. 

Are you running windows or Linux?
Does db.log contain any information ?
can you post a pastebin.com link of the last few hundred lines of debug.log
if your running windows can you pastebin.com the last 100 lines of your debug2.log
if it is a crash in windows (the application just closes) was there any pop up error and if not can you review in eventvwr the data related to the crash and post that as well.

Any other information would be helpful as well

**Comment by denravonska** [comment]
Does it crash or go offline? Or are these two separate issues?

**Comment by AndreasMager** [comment]
I'am running Windows 10.

20:38 was my last POR
20:47 the crash.

db.log is empty.
debug.log: https://pastebin.com/p1pzqNyR
debug2.log contains no information anymore. 

The application has exited. No Popup for debug.
I wasn't present during the event.

**Comment by iFoggz** [comment]
I don't see any issues in debug.log. How are your updating your client? In wallet or msi installer? I see lots of signing blocks which is normal but the output of those are far less or hidden now I believe (I'll double check). Can u load up eventvwr in start -> run (Windowskey  + r) and look for the crash event.

**Comment by AndreasMager** [comment]
It crasched again but this time without a  Por event.
eventvwr:
```
- <Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
- <System>
  <Provider Name=".NET Runtime" /> 
  <EventID Qualifiers="0">1026</EventID> 
  <Level>2</Level> 
  <Task>0</Task> 
  <Keywords>0x80000000000000</Keywords> 
  <TimeCreated SystemTime="2017-07-28T20:51:13.569400800Z" /> 
  <EventRecordID>1368123</EventRecordID> 
  <Channel>Application</Channel> 
  <Computer>DESKTOP-5CFUEDB</Computer> 
  <Security /> 
  </System>
- <EventData>
  <Data>Anwendung: gridcoinresearch.exe Frameworkversion: v4.0.30319 Beschreibung: Der Prozess wurde aufgrund einer unbehandelten Ausnahme beendet. Ausnahmeinformationen: Ausnahmecode c0000005, Ausnahmeadresse 004BBE53 Stapel:</Data> 
  </EventData>
  </Event>

- <Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
- <System>
  <Provider Name=".NET Runtime" /> 
  <EventID Qualifiers="0">1026</EventID> 
  <Level>2</Level> 
  <Task>0</Task> 
  <Keywords>0x80000000000000</Keywords> 
  <TimeCreated SystemTime="2017-07-29T15:48:40.953545500Z" /> 
  <EventRecordID>1368631</EventRecordID> 
  <Channel>Application</Channel> 
  <Computer>DESKTOP-5CFUEDB</Computer> 
  <Security /> 
  </System>
- <EventData>
  <Data>Anwendung: gridcoinresearch.exe Frameworkversion: v4.0.30319 Beschreibung: Der Prozess wurde aufgrund einer unbehandelten Ausnahme beendet. Ausnahmeinformationen: Ausnahmecode c0000005, Ausnahmeadresse 004BBE53 Stapel:</Data> 
  </EventData>
  </Event>
```

**Comment by tomasbrod** [comment]
Does this still occur with the latest version? @AndreasMager

**Comment by AndreasMager** [comment]
Sorry i have stopped running the wallet 24/7. My power bill exploded :D .  I noticed no crash in the run time slots.

**Comment by denravonska** [comment]
This looks like an NN sync crash. In that case it should be fixed in 3.7.0.0.
